include/audio/aseq.h

```diff
@@ -0,0 +1,2341 @@
+    .endm
+
+    /**
+     * unk_A7 <arg:u8>
+     *
+     *  TODO DESCRIPTION
+     */
+    .macro unk_A7 arg
+        _wr_cmd_id  unk_A7, ,0xA7,,,,,,, 0, 0
+        _wr_u8      \arg
+    .endm
+
+    /**
+     * unk_A6 <arg0:u8> <arg1:s16>
+     *
+     *  TODO DESCRIPTION
+     */
+    .macro unk_A6 arg0, arg1
+        _wr_cmd_id  unk_A6, ,0xA6,,,,,,, 0, 0
+        _wr_u8      \arg0
+        _wr_s16     \arg1
+    .endm
+
+    /**
+     * unk_A5
+     *
+     *  TODO DESCRIPTION
+     */
+    .macro unk_A5
+        _wr_cmd_id  unk_A5, ,0xA5,,,,,,, 0, 0
+    .endm
+
+    /**
+     * unk_A4 <arg:u8>
+     *
+     *  TODO DESCRIPTION
+     */
+    .macro unk_A4 arg
+        _wr_cmd_id  unk_A4, ,0xA4,,,,,,, 0, 0
+        _wr_u8      \arg
+    .endm
+
+    /**
+     * unk_A3
+     *
+     *  TODO DESCRIPTION
+     */
+    .macro unk_A3
+        _wr_cmd_id  unk_A3, ,0xA3,,,,,,, 0, 0
+    .endm
+
+    /**
+     * unk_A2 <arg:s16>
+     *
+     *  TODO DESCRIPTION
+     */
+    .macro unk_A2 arg
+        _wr_cmd_id  unk_A2, ,0xA2,,,,,,, 0, 0
+        _wr_s16     \arg
+    .endm
+
+    /**
+     * unk_A1
+     *
+     *  TODO DESCRIPTION
+     */
+    .macro unk_A1
+        _wr_cmd_id  unk_A1, ,0xA1,,,,,,, 0, 0
+    .endm
+
+    /**
+     * unk_A0 <arg:s16>
+     *
+     *  TODO DESCRIPTION
+     */
+    .macro unk_A0 arg
+        _wr_cmd_id  unk_A0, ,0xA0,,,,,,, 0, 0
+        _wr_s16     \arg
+    .endm
+
+#endif
+
+/**
+ * ptradd <value:lbl>
+ *
+ *  Computes TP += value
+ */
+.macro ptradd value
+    _wr_cmd_id  ptradd, ,0xBC,,,,,,, 0, 0
+    _wr_lbl     \value
+.endm
+
+/**
+ * ptraddi <value:u16>
+ *
+ *  Like ptradd but for immediates instead of labels
+ *
+ *  Computes TP += value
+ */
+.macro ptraddi value
+    _wr_cmd_id  ptradd, ,0xBC,,,,,,, 0, 0
+    _wr_u16     \value
+.endm
+
+/**
+ * combfilter <arg0:u8> <arg1:u16>
+ *
+ *  Enable comb filter.
+ *  TODO args? arg0=16,arg1=val<<8 maps well to midi chorus
+ */
+.macro combfilter arg0, arg1
+    _wr_cmd_id  combfilter, ,0xBB,,,,,,, 0, 0
+    _wr_u8      \arg0
+    _wr_u16     \arg1
+.endm
+
+/**
+ * randgate <range:u8>
+ *
+ *  Sets the range of random note gateTime fluctuations.
+ *
+ *  NOTE: This feature is bugged. If this is non-zero it wll actually use the range set by randvel.
+ */
+.macro randgate range
+    _wr_cmd_id  randgate, ,0xBA,,,,,,, 0, 0
+    _wr_u8      \range
+.endm
+
+/**
+ * randvel <range:u8>
+ *
+ *  Sets the range for random note velocity fluctuations.
+ */
+.macro randvel range
+    _wr_cmd_id  randvel, ,0xB9,,,,,,, 0, 0
+    _wr_u8      \range
+.endm
+
+/**
+ * rand <max:u16>
+ *
+ *  Stores a random number in the range [0, max) into TP. If max is 0 the range is [0, 65535]
+ */
+.macro randtoptr max
+    _wr_cmd_id  randtoptr, ,0xB7,,,,,,, 0, 0
+    _wr_u16     \max
+.endm
+
+/**
+ * dyntblv
+ *
+ *  Loads DYNTBL[TR] -> TR
+ */
+.macro dyntblv
+    _wr_cmd_id  dyntblv, ,0xB6,,,,,,, 0, 0
+.endm
+
+/**
+ * dyntbltoptr
+ *
+ *  Loads DYNTBL[TR] -> TP
```

Comment:
```suggestion
 *  Loads ((u16*)DYNTBL)[TR] -> TP
```
https://github.com/zeldaret/oot/blob/65cc42a71737e945fa8f7d3917aaae2efc474e0d/src/audio/lib/seqplayer.c#L1574
I'm confused why this implementation is different from `dyntbllookup`. It does the same, right?